tests/sentry/dynamic_sampling/tasks/test_tasks.py

```diff
@@ -288,7 +288,8 @@ def test_config_invalidation_when_sample_rates_do_not_change(
     ):
         extrapolate_monthly_volume.side_effect = self.forecasted_volume_side_effect
         get_transaction_sampling_tier_for_volume.side_effect = self.sampling_tier_side_effect
-        get_blended_sample_rate.return_value = 0.8
+        get_blended_sample_rate.return_value = 1.0
```

Comment:
The test was now correctly fixed, before it was succeeding but because there was an early return, thus it was a false positive.